code-samples/notification.ts

```diff
@@ -6,3 +6,12 @@ import { Notification } from '@nulib/design-system';
     <p>I want to tell you something</p>
 </Notification>
 `;
+
+export const notificationWithDelete = `
+// Supports a delete button, by applying "delete" class to child element
+
+<Notification>
+    <button className="delete" onClick={() => console.log("remove")} />
```

Comment:
@adamjarling Is this PR just introduced the delete concept or should it be fully functional? Currently seems to just console.log().

Reply:
@mathewjordan This is just a usage example.  The `Notification` component as used in Meadow, wraps whatever JSX you want to provide as `children`.  `Notification` currently doesn't support any type of delete concept and thinking more about it, maybe it shouldn't?  Otherwise we'll have to provide a callback `prop` to Notification to handle the delete icon click.  Every app could do something different with that click, deciding on how or when to render the `Notification`.

Reply:
What if it was handled by a prop `isCloseable` with a default to false and the button itself is behind the scenes? That might be a clean way to use it when brought into other packages?

` <Notification isCloseable> The textual stuff. </Notification>`

Reply:
Either way, I'm good to introduce this now and we can refine later, cool with that?

Reply:
Yea, maybe that's the way to go.  We give it a prop like `isClosable`, and if the user closes the notification, it goes bye-bye and that's it.  The use case for the screen shot above, uploading a file, is perhaps not the best usage of the notification close button to fire a custom callback function rather than just closing the notification.  Maybe that UI should have an explicit button or icon that triggers clearing the selected file.

So, I'll run with `isClosable` and see how it goes.